src/observer.ts

```diff
@@ -52,6 +57,22 @@ export function observer<P extends object, TRef = {}>(
         memoComponent = memo(wrappedComponent)
     }
 
-    memoComponent.displayName = baseComponentName
+    memoComponent.displayName = `observer(${baseComponentName})`
```

Comment:
I'm not entirely sure about this change; it seems kinda consistent, since `observer` is basically a hook, (it was already a feature request on mobx-react 5 as well). On the other hand, one could also see from the hooks in the inspector that something is an observer. And it would liter `observer(XXX)` all over the place

Reply:
Yea, I think we don't really need it. As you say, it will be visible in hooks inspector and it's not like people need to know if a component is observed all the time. I would remove it.

Reply:
If it has to be there I'd rather see it as `Component (observer)`

Reply:
Agreed, I'll drop it

Reply:
Dropped